src/file/file.service.ts

```diff
@@ -90,9 +90,30 @@ export class FileService {
    * @param fileId - the id of the file.
    * @param partialFile - the partial file.
    */
-  public static updateById(fileId: string, partialFile: Partial<IFile>): Promise<IFile> {
+  public static updateById(fileId: string, partialFile: Partial<IFile>): Promise<boolean> {
     return FilesRepository.updateById(fileId, partialFile);
-  }
+	}
+	
+	/**
+	 * updateMany updates a list of files.
+	 * @param files - List of files to update with their updated fields and their id.
+	 */
+	static async updateMany(files: (Partial<IFile> & { id: string })[]): Promise<{updated: string[], failed: { id: string, error: Error }[]}> {
+		const failedFiles: { id: string, error: Error }[] = [];
+		const updatedFiles: string[] = [];
+		for (let i = 0; i < files.length; i++) {
+			try {
+				const updatedFile = await FilesRepository.updateById(files[i].id, files[i]);
+				if (updatedFile) {
+					updatedFiles.push(files[i].id);
+				}
+			} catch (e) {
```

Comment:
change it to 'error', this is not go

Reply:
How is go relevant to the variable being named `e`, if it was go then I would name it `err`, and in JS the standard is `e` but `error`, `err`, `ex` and `exception` are as fitting for it as the rest of the naming options. There's no point in renaming a variable unless it makes it clearer what it is, and the only variable that can be passed to a `catch` statement is the exception that was caught.